### PR TITLE
Add german translations

### PR DESCRIPTION
**Pull request by System98**
Add missing german translations to Armory, Bodybag, Radios and
Shootingrange

### CODE REVIEW

File: addons/armory/stringtable.xml [hunk 1]
```diff
@@ -114,6 +115,8 @@
         </Key>
         <Key ID="STR_TAC_Armory_ShowOnlyCompatible">
             <English>Show only items compatible with equipped weapons.</English>
+            <German>Zeige nur Gegenstände, die mit ausgerüsteten Waffen kompatibel sind</German>
+            <Italian>.</Italian>
```
Comment (jonpas) [inline addons/armory/stringtable.xml]: That probably shouldn't be like that.

Comment (System98) [inline addons/armory/stringtable.xml]: That definatly shouldn't be like that.

File: addons/armory/stringtable.xml [hunk 2]
```diff
@@ -114,6 +115,7 @@
         </Key>
         <Key ID="STR_TAC_Armory_ShowOnlyCompatible">
             <English>Show only items compatible with equipped weapons.</English>
+            <German>Zeige nur Gegenstände, die mit ausgerüsteten Waffen kompatibel sind</German>
```
Comment (jonpas) [inline addons/armory/stringtable.xml]: Now there is a missing point at the end which was under Italian before. :laughing:

Comment (System98) [inline addons/armory/stringtable.xml]: God dammit! :facepalm:

**Comment by jonpas** [comment]
Thank you! :+1:

**Pull request merged**